lib/internal/freelist.js

```diff
@@ -23,4 +25,14 @@ class FreeList {
   }
 }
 
-module.exports = FreeList;
+function setIsReused(item, reused) {
+  item[is_reused_symbol] = reused;
+  return item;
+}
+
+module.exports = {
+  FreeList,
+  symbols: {
+    is_reused_symbol
+  }
```

Comment:
I think we are placing the symbols at top level anyway, even though I do not dislike this pattern.

Reply:
I borrowed this pattern from lib/internal/async_hooks.js. I'm not sure I understand what you would like to see here instead? Could you elaborate?

Reply:
Oh don't worry about it.